cl/scrapers/management/commands/clone_from_cl.py

```diff
@@ -0,0 +1,406 @@
+"""
+This tool allows you to partially clone data from courtlistener.com to your
+local environment, you only need to pass the type and object id and run it.
+
+manage.py clone_from_cl --type search.Opinion --id 9355884
+manage.py clone_from_cl --type search.Docket --id 5377675
+manage.py clone_from_cl --type people_db.Person --id 16207
+manage.py clone_from_cl --type search,Court --id usnmcmilrev
+
+This tool is only for development purposes, so it only works when
+the DEVELOPMENT env is set to True. It also relies on the CL_API_TOKEN
+env variable.
+
+You can also pass the api token before running the command:
+
+CL_API_TOKEN='my_api_key' manage.py clone_from_cl --type search.Opinion --id 9355884
+
+You can also clone multiple objects at the same time, for example:
+
+manage.py clone_from_cl --type search.OpinionCluster --id 1867834 1867833
+manage.py clone_from_cl --type search.Docket --id 14614371 5377675
+manage.py clone_from_cl --type search.Court --id mspb leechojibtr
+manage.py clone_from_cl --type people_db.Person --id 16212 16211
+
+This is still work in progress, some data is not cloned yet.
+"""
+
+import os
+
+import requests
+from django.apps import apps
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.core.management import BaseCommand
+from django.db import IntegrityError, transaction
+from django.urls import reverse
+from django.utils.dateparse import parse_date
+from requests import Session
+
+from cl.search.models import Citation, Opinion
+from cl.search.tasks import add_items_to_solr
+
+VALID_TYPES = (
+    "search.OpinionCluster",
+    "search.Docket",
+    "people_db.Person",
+    "search.Court",
+)
+
+cluster_endpoint = "https://www.courtlistener.com/api/rest/v3/clusters/"
+dockets_endpoint = "https://www.courtlistener.com/api/rest/v3/dockets/"
+people_endpoint = "https://www.courtlistener.com/api/rest/v3/people/"
+courts_endpoint = "https://www.courtlistener.com/api/rest/v3/courts/"
+
+
+def clone_opinion_cluster(
+    session: Session, cluster_ids: list, object_type="search.OpinionCluster"
+):
+    """
+    Download opinion cluster data from courtlistener.com and add it to
+    local environment
+    :param session: a Requests session
+    :param cluster_ids: a list of opinion cluster ids
+    :param object_type: OpinionCluster app name with model name
+    :return: list of opinion cluster objects
+    """
+
+    opinion_clusters = []
+
+    for cluster_id in cluster_ids:
+        print(f">> Cloning opinion cluster id: {cluster_id}")
+        model = apps.get_model(object_type)
```

Comment:
It's not really necessary, but i did it like this so that all clone functions are consistent

Reply:
I don't see anywhere that you *couldn't* remove this though, right? You can have them consistent _and_ have none of them pass the arg?

Reply:
I did it this way to accept the object type arg as an input and use it to get the model, the argument could be removed from all of them but then i'll have to import all of the models manually and the object type input would be only a string to select which function to call. 

The other way is to remove only the object_type argument only in clone_opinion_cluster function, and let the others to  use the default object type value in case of being called from clone_opinion_cluster function or use the object type input arg in case you call the other functions individually(clone only dockets, courts or people) 

what do you think?